extra/1-currency-conversion.js

```diff
@@ -5,8 +5,11 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
-
+function convertToUSD(convertGBP) {
+  let rate = 1.4;
+  let convertToUSD = convertGBP * rate;
+  return convertToUSD;
+}
```

Comment:
hi, technically this function work ok, it does when it should do. 
however, there are still couple things that might be done even better in it, following best practices.
1. argument that function receives is named as `convertGBP`, but word "convert" is a verb and means action, and so this name better suits for a function name rather than for a variable of argument. Better name would be `priceGBP`.
2. similar logic as above applied to the result that we return from the functions. It's rather `priceUSD`.
3. finally, if your function always converts GBP to USD you can make it even more simple, like this:
```js
function convertToUSD(priceGBP) {
  return priceGBP * 1.4;
}
```